### DISCUSSION TITLE
WebP Preview in file browser

### DISCUSSION THREAD

**Discussion by renderfarmer** [2sic/2sxc#2712] [Ideas]
Hi there, we noticed that Webp Images are not displayed in Adam/2SXC File manager like other Images are. You can upload them and then use them, but this is kind of difficult for unexperienced people using the system. So my question is, what would it take to implement a WebP Image Preview in 2SXC so you can work with them as with any other image files.

Thanks

**Comment by iJungleboy** [comment]
We must look into this, shouldn't be difficult.
